sdk-extensions/logging/src/main/java/io/opentelemetry/sdk/logging/data/LogRecord.java

```diff
@@ -32,9 +36,23 @@ static LogRecord create(
       AnyValue body,
```

Comment:
I would love a follow-on to convert this to a `String`.

Reply:
Sure, let me know if you want it in this PR, or is it fine to modify this on our next PR?

Reply:
a follow-on PR is fine.

Reply:
Why are we deviating from the log model here?